### PR TITLE
0.8.0 doc updates

### PR DESCRIPTION
**Pull request by yadudoc**
Updates configs for:
* Comet
* Cori
* Stampede2
* Theta
* CC-IN2P3
* AWS

### CODE REVIEW

File: parsl/configs/ec2_htex_single_node.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from parsl.executors import HighThroughputExecutor
+from parsl.addresses import address_by_query
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label='ec2_single_node',
+            address=address_by_query(),
+            provider=AWSProvider(
+                image_id='YOUR_AMI_ID',     # Please replace image_id with your image id, e.g., 'ami-82f4dae7'
```
Comment (annawoodard) [inline parsl/configs/ec2_htex_single_node.py]: Remove 'please's, consider removing comments entirely, they aren't adding any info

File: parsl/configs/theta_local_htex_multinode.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            max_workers=4,
+            address=address_by_hostname(),
+            provider=CobaltProvider(
+                queue="debug-flat-quad",
+                launcher=AprunLauncher(),
+                walltime="00:30:00",
+                nodes_per_block=2,
+                init_blocks=1,
+                max_blocks=1,
+                scheduler_options='',     # Input your scheduler_options if needed
```
Comment (annawoodard) [inline parsl/configs/theta_local_htex_multinode.py]: This isn't very informative, I favor a description of scheduler_options instead, for example:
```
# string to prepend to #SBATCH blocks in the submit script to the scheduler
```

File: parsl/configs/ec2_htex_single_node.py (hunk 1 above)
Comment (yadudoc) [reply parsl/configs/ec2_htex_single_node.py]: Fixed.

File: parsl/configs/theta_local_htex_multinode.py (hunk 2 above)
Comment (yadudoc) [reply parsl/configs/theta_local_htex_multinode.py]: Agreed. I've updated all configs from my test group. Will update the rest once they are merged.

**Pull request merged**